lab2/Pasichnichenko_Task2.py

```diff
@@ -0,0 +1,11 @@
+# Multi-paradigm programming languages Lab2 Illia Pasichnichenko IKM-221a
+print('Multi-paradigm programming languages Lab2')
+print('Illia Pasichnichenko IKM-221a')
+
+x = float(input("Enter x value: "))
+y = float(input("Enter y value: "))
+z = float(input("Enter z value: "))
+
+result = (11 + 2 * x + 4.1) / (12.4 - y) + z
+
+print(result)
```

Comment:
It looks like you broke the pipeline for creating new branches.
please recall me to discuss it